libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1158,12 +1181,97 @@ def check_pods_status(node_obj) -> bool:
+                        assert_utils.assert_true(resp[0], resp[1])
+        return True, service_status[-1]
+
+    def client_config(self, master_node_list, namespace):
+        """
+        This method is used to setup the client
+        param: master_node_list: master_node_obj
+        param: namespace: custom namespace
+        returns True, s3t_obj, list of access,secret ksy with ext_port_ip
+        """
+        LOGGER.info("Setting the current namespace")
+        resp_ns = master_node_list[0].execute_cmd(
+            cmd=common_cmd.KUBECTL_SET_CONTEXT.format(namespace),
+            read_lines=True)
+        LOGGER.debug("response is %s,", resp_ns)
+        resp = system_utils.execute_cmd(
+            common_cmd.CMD_GET_IP_IFACE.format(self.deploy_cfg['iface']))
+        eth1_ip = resp[1].strip("'\\n'b'")
+        if self.service_type == "NodePort":
+            resp = ext_lbconfig_utils.configure_nodeport_lb(master_node_list[0],
+                                                            self.deploy_cfg['iface'])
+            if not resp[0]:
+                LOGGER.debug("Did not get expected response: %s", resp)
+            ext_ip = resp[1]
+            port = resp[2]
+            ext_port_ip = Template(self.deploy_cfg['https_protocol']
+                                   + ":$port").substitute(ip=ext_ip, port=port)
+            LOGGER.debug("External LB value, ip and port will be: %s", ext_port_ip)
+        else:
+            LOGGER.info("Configure HAproxy on client")
+            ext_lbconfig_utils.configure_haproxy_rgwlb(master_node_list[0].hostname,
+                                                       master_node_list[0].username,
+                                                       master_node_list[0].password,
+                                                       eth1_ip, self.deploy_cfg['iface'])
+            ext_port_ip = Template(self.deploy_cfg['https_protocol']).substitute(ip=
+                                                                                 eth1_ip)
+        LOGGER.info("Step to Create S3 account and configure credentials")
+        if self.s3_engine == 2:  # "s3_engine flag is used for picking up the configuration
+            # for legacy s3 and rgw, `1` - legacy s3 and `2` - rgw"
+            resp = self.post_deployment_steps_lc(self.s3_engine, ext_port_ip)
+            assert_utils.assert_true(resp[0], resp[1])
+            access_key, secret_key = S3H_OBJ.get_local_keys()
+            if self.service_type == "NodePort":
+                s3t_obj = S3TestLib(access_key=access_key, secret_key=secret_key,
+                                    endpoint_url=ext_port_ip)
+            else:
+                s3t_obj = S3TestLib(access_key=access_key, secret_key=secret_key)
+            response = [access_key, secret_key, ext_port_ip]
+        return True, s3t_obj, response
+
+    # pylint: disable=too-many-arguments,too-many-locals,too-many-branches,too-many-statements
```

Comment:
do we want to add pylint disables seperate on each line for better readability purpose?